streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -951,6 +956,12 @@ public StreamsMetricsImpl(Metrics metrics) {
 
             this.taskDestructionSensor = metrics.sensor(sensorNamePrefix + ".task-destruction");
             this.taskDestructionSensor.add(metrics.metricName("task-destruction-rate", metricGrpName, "The average per-second number of destructed tasks", metricTags), new Rate(new Count()));
+
+            this.processedRecordsSensor = metrics.sensor(sensorNamePrefix + ".processed-records");
```

Comment:
Personally I'd suggest we only keep a skip sensor for user to monitor correctness since the the "processed record sensor" is well covered in the throughput sensor already. And for that skip sensor we can use `Rate()`. For example, as in `record-error-rate` in `Sender.class`.